packages/slate-react/src/components/leaf.tsx

```diff
@@ -43,7 +46,13 @@ const Leaf = (props: {
   }, [placeholderRef])
 
   let children = (
-    <String isLast={isLast} leaf={leaf} parent={parent} text={text} />
+    <String
+      key={keyForString++}
```

Comment:
Increase key on every leaf render may potentionally burden the performance?

Reply:
yes, but I don't find any other better ways